container/examples/container-wc/myWebComponent.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * This class is used to test LuigiClient webcomponent based functionality
+ */
+export default class extends HTMLElement {
+  constructor() {
+    super();
+    const shadowRoot = this.attachShadow({ mode: 'open' });
+    const template = document.createElement('template');
+    template.innerHTML = `<section><p id="paragraph">Hello World !!! </p></section>`;
```

Comment:
```suggestion
    template.innerHTML = `<section><p style="border: solid blue 2px;" id="paragraph">Hello World !!! </p></section>`;
```

To make it visually easy for the client trying it out, to see where the webcomponent is.